lockfile/src/csharp.rs

```diff
@@ -143,9 +147,11 @@ impl Parse for CSProj {
     /// Parses `.csproj` files into a vec of packages
     fn parse(&self, data: &str) -> anyhow::Result<Vec<Package>> {
         let data = data.trim_start_matches(UTF8_BOM);
-        let mut de =
-            Deserializer::new_from_reader(data.as_bytes()).non_contiguous_seq_elements(true);
-        let parsed = Project::deserialize(&mut de)?;
+        // let mut de =
+        // Deserializer::new_from_reader(data.as_bytes()).
+        // non_contiguous_seq_elements(true); let parsed =
+        // Project::deserialize(&mut de)?;
```

Comment:
I'd assume this patch is a WIP, please feel free to re-request a review once it's done.

Reply:
Forgot to cleanup. Fixed now